[web]/ajax/ajax.js

```diff
@@ -115,18 +115,16 @@ function getKeyCode ( e )
     return keynum;
 }
 
-var elementManager = new ElementManager();
-
-function ElementManager() {
-	this.elements = new Array();
-	this.get = function (id) {
-		for ( i = 0; i < this.elements.length; i++ )
-		{
+class ElementManager {
+	constructor() {
+		this.elements = new Array();
+	}
+	
+	get(id) {
+		for (let i = 0; i < this.elements.length; i++) {
 			var element = this.elements[i];

```

Comment:
Prefer [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of).

```suggestion
		for (const element of this.elements) {

```